### ISSUE TITLE
Support multiple entry points

### ISSUE THREAD

**Issue by oleg-andreyev** [ai/size-limit#30]
I my project we have multiple entry points configured with https://webpack.js.org/configuration/output/#output-filename

but when running size-limit, getting following issues:
```
ERROR  TypeError: Cannot read property 'size' of undefined
    at runWebpack.then.stats (/node_modules/size-limit/index.js:166:53)
```

because 

```
let name = `${ stats.compilation.outputOptions.filename }` // filename template, before replaced placeholder https://webpack.js.org/configuration/output/#output-filename
      name += opts.config ? '' : '.gz'
      const assets = stats.toJson().assets

      const size = assets.find(i => i.name === name).size
```

**Comment by ai** [comment]
Do you use custom webpack config? Can you show your Size Limit config?

Do you use Size Limit for open source library or application?

**Comment by oleg-andreyev** [comment]
@ai yes we're using custom webpack config (I was waiting for #29), Size Limit configs are the default ones:

```
"size-limit": [
    {
      "path": "index.js",
      "limit": "9 KB"
    }
  ]
```

Using it for application.

**Comment by ai** [comment]
Honestly, the best way to use Size Limit for application is `webpack: false` https://github.com/ai/size-limit#applications

**Comment by SivanMehta** [comment]
@ai @oleg-andreyev I have attempted to fix this error in #31. I was finding this bug as well as I was attempting to use `size-limit@0.12.0`. Any feedback would be great!

**Comment by ai** [comment]
Released in 0.12.1

@oleg-andreyev but you should to add Webpack Bundle Analyzer and set `webpack: false` to Size Limti. Custom Configs is not for your case.
